dapps/marketplace/public/translations/de_DE.json

```diff
@@ -247,7 +248,7 @@
   "VUfcH": "Promote Origin",
   "9rfJX": "Requires: Twitter attestation",
   "1cVxL1": "Follow Origin",
-  "2AoW77": "Requires: Twitter attestation",
+  "49fgup": "Requires: Twitter and Telegram attestation",
```

Comment:
Shouldn't this be "Twitter or Telegram attestation" ?
That string comes from here: https://github.com/OriginProtocol/origin/blob/master/dapps/marketplace/src/components/growth/ActionGroup.js#L48

Or maybe we don't need to lock the Follow section at all since it also has Facebook and no attestations are need for Facebook follow...

@sparrowDom Any input ?